extensions/wikia/VideosModule/Modules/VideosModuleStaff.class.php

```diff
@@ -38,7 +38,7 @@ public function getCacheKey() {
 		sort( $category );
 		$hashCategory = md5( json_encode( $category ) );
 
-		return implode( ':', $cacheKey, $hashCategory, $this->sort );
+		return implode( ':', [$cacheKey, $hashCategory, $this->sort] );
```

Comment:
Good point @jsutterfield I found more instances of this problem and fixed them too.

>  does this address the original issue of empty results for the Videos Module?

Yes, this was basically causing the return value for implode to be `null` which made the cache key to be null and therefore always returning empty results for videosModule. You can verify this on your devbox before and after.